crates/re_renderer/src/size.rs

```diff
@@ -32,27 +32,26 @@ impl Size {
     /// Values passed must be finite positive.
     #[inline]
     pub fn new_points(size: f32) -> Self {
-        debug_assert!(size.is_finite() && size >= 0.0, "Bad size: {size}");
+        debug_assert!((0.0..f32::MAX).contains(&size), "Bad size: {size}");
         Self(-size)
     }
 
     /// Returns true if the size is an automatically determined size ([`Self::AUTO`] or [`Self::AUTO_LARGE`]).
     #[inline]
     pub fn is_auto(&self) -> bool {
-        self.0.is_infinite()
+        self.0 >= f32::MAX || self.0 <= f32::MIN
```

Comment:
number-line ordering:

```suggestion
        self.0 <= f32::MIN || f32::MAX <= self.0
```